app/lib/menu/menu-builder.js

```diff
@@ -268,6 +269,17 @@ class MenuBuilder {
     return this;
   }
 
+  appendNewProcessApplication() {
+    this.menu.append(new MenuItem({
+      label: 'New Process Application',
```

Comment:
```suggestion
      label: 'New Process Application...',
```
The dots suggest that it opens a new window.